docs/editions.rst

```diff
@@ -28,40 +28,60 @@ limitations.
 
 .. _standard-edition:
 
-CrateDB
-=======
+CrateDB Standard Edition
```

Comment:
This was intentionally just called "CrateDB". There is some discussion whether this should be called "CrateDB Enterprise Edition"

cc: @joemoe

Reply:
At the moment it is `CrateDB`.
Might change, but that's how it is at the moment.